adafruit_ads1x15/ads1x15.py

```diff
@@ -170,8 +174,12 @@ def _read(self, pin):
         self._write_register(_ADS1X15_POINTER_CONFIG, config)
 
         if self.mode == Mode.SINGLE:
+            # poll conversion complete status bit
             while not self._conversion_complete():
                 pass
+        else:
+            # just sleep (can't poll in continuous)
+            time.sleep(2 / self.data_rate)
```

Comment:
According to the ADS1115 data sheet (and also the ADS1015) the conversion time is 1 / DR:

> 9.3.6 Output Data Rate and Conversion Time
> The ADS111x offer programmable output data rates. Use the DR[2:0] bits in the Config register to select output
> data rates of 8 SPS, 16 SPS, 32 SPS, 64 SPS, 128 SPS, 250 SPS, 475 SPS, or 860 SPS.
> Conversions in the ADS111x settle within a single cycle; thus, the conversion time is equal to 1 / DR.
Any particular reason you chose 2 / DR?

Reply:
I tried 1 / DR first and it didn't seem to work. Might have something to do with extra settling time needed due to the mux being changed? Not sure.